test/features/lb/public_lb.feature

```diff
@@ -46,10 +47,10 @@ Feature: Test different tfvars configurations for the lb module with a public ti
             | action | resource                | name        | count |
             #--------|-------------------------|-------------|-------|
             | create | aws_lb                  | default     |       |
-            |        | aws_lb_listener         | default     | 2     |
+            |        | aws_lb_listener         | default     |       |
             |        | aws_lb_target_group     | default     |       |
             |        | aws_security_group      | default     |       |
-            |        | aws_security_group_rule | internet_in | 2     |
+            |        | aws_security_group_rule | internet_in |       |
```

Comment:
Why did the count change?

Reply:
According to the return data, only one instance was being created. If this is not the case, then there are 2 possibilities:

1. The return data that we get from the 3rd party terraform call is not accurate
2. For lack of a better word, there's a transliteration error when remapping the data